imports/plugins/core/ui/client/containers/buttonSelectContainer.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component, PropTypes } from "react";
+import classnames from "classnames";
+import { composeWithTracker, merge } from "/lib/api/compose";
+import Button from "../components/button/button.jsx";
+import { ButtonSelect } from "../components/button";
+
+class ButtonSelectContainer extends Component {
```

Comment:
Container components should only contain business logic and no manipulation of the view. The buttonSelect component should take what ever props you pass in, and do all the necessary rendering.